### PR TITLE
index.html file was modfied style.css file was modified and all linte…

### PR DESCRIPTION
**Pull request by asadmisrani**
All linter errors have been solved successfully.

### CODE REVIEW

**Review by divinecharlotte — CHANGES_REQUESTED** [review]
Hi @asadmisrani  , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 💯 
- No linters errors ✔️ 



## Required Changes ♻️
☠️ next time consider submitting a Pr that contains all your changes which will help reviewers to give you inline feedback and that will help you to understand easily where and why there required changes
- [ ] kindly consider adding Button interactions such as hover, active (when it's pressed), disabled, and enable (default state) are styled as shown in the Figma template. this will improve the user experience
- you may choose any from the below 👇 
opd

_Check the comments under the review._
## Optional suggestions

- [ ] [OPTIONAL]kindly consider adding more description to your PR to make it more descriptive and professional

- [ ] [OPTIONAL]kindly consider adding meaningful commit messages this will help you in the future once you want to revert your code like in the case a bug is introduced and you want to check out on the commit where the bug was not yet introduced

- [ ][OPTIONAL] kindly consider editing this section 👇 
kk

by adding related information to this current project that will make your readme look more professional 😃 

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by asadmisrani** [comment]
all changes have been made.

**Comment by asadmisrani** [comment]
All the required changes have been made.

File: index.html [hunk 1]
```diff
@@ -232,7 +232,7 @@ <h2>Skills</h2>
       <input type="text" id="name" name="user_name" placeholder="Full name" maxlength='30' required />
       <input type="email" name="user_email" id="email" placeholder="Email address" required />
       <textarea name="comment" id="comment" placeholder='Enter text here' maxlength='500' required></textarea>
-      <button type="submit">Get in Touch</button>
+      <button type="submit" class="my-button" disabled>Get in Touch</button>
     </form>
```
Comment (Gambit142) [inline index.html]: - [ ] The previous reviewer did not ask that you disable the button. The reviewer asked that you have style declarations for your `hover`, `active` and `disable` states for your buttons. Kindly remove the `disabled` attribute you added to this button. I can see that you have the declarations for the `hover` and `active` states. However, the style declaration for the `hover` is missing the `rgba` keyword. Kindly include it on line 240 of your stylesheet; `box-shadow: 0 8px 16px rgba(54, 179, 127, 0.24);`

Also, kindly add the `disabled` style declaration for the button. You can read this [stack overflow](https://stackoverflow.com/questions/14750078/style-disabled-button-with-css) article to see how to include the disabled style declaration for buttons 👍 💻

**Review by Gambit142 — CHANGES_REQUESTED** [review]
Hi @asadmisrani @Joaquin123455 👋, 

Great work on making the changes requested by a previous reviewer 👏 

## Highlights
- All linters checks passed ✔️ 
- Use of correct HTML validations ✔️ 

You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.

 ## Required Changes ♻️
_Kindly check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### Please, remember to tag me in your question so I can receive the notification. You can tag me by typing @Gambit142

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by asadmisrani** [comment]
Dear Sir,
I have deleted the hover and active button status because it is not part of
this Portfolio: add a contact form and finish the mobile version, I made
this by mistake. you can see the actual project's
general requirements below. I apologize for every mistake I have made in
this project which was by mistake.

### General requirements

- Make sure that there are [no linter
errors](https://github.com/microverseinc/linters-config).
- Make sure that you used correct [GitHub
flow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md).
- Make sure that you documented your work [in a professional
way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md).

### HTML/CSS requirements

- Follow our list of [best practices for HTML &
CSS](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).

### Project requirements

_**IMPORTANT NOTE:** For each Microverse project, you need to create a
separate pull request. [Read how to work with multiple pull requests
in your one repository in the most efficient
way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/multiple_pull_requests.md)._

- The section you need to build for this milestone is:
    - the contact form:
    <p align="center">
     Form
    </p>
    - and the footer (if present in the design)
We included a screenshot of Template 1 for reference, but you should
follow the template that you chose.
- Make sure that:
    - Name field accepts up to 30 chars.
    - Email field accepts only emails.
    - Message field accepts up to 500 chars.
    - All fields have placeholder text.
    - All fields have the correct type.
    - A user cannot submit the form if one or more of the fields is empty.
- Collect data by using the [Formspree service](https://formspree.io/html).
- In order to lay out the elements on the page you should use Flexbox.
- You don't need to implement any functionality that requires JavaScript.
- You must stick to the design as much as possible (e.g., font,
colors, images, text, margins) using [the templates in
Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).
- Implement the button interactions (enable, hover, etc.).

**Review by iambenkis — APPROVED** [review]
Hi @asadmisrani  , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

Linters are passing :heavy_check_mark: 
The project is running successfully :heavy_check_mark: 
PR is well descriptive :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@iambenkis** in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by asadmisrani** [comment]
Thanks a lot.

**Pull request merged**